packages/collector/test/tracing/protocols/graphql/test.js

```diff
@@ -24,135 +24,155 @@ const globalAgent = require('../../../globalAgent');
 
 const agentControls = globalAgent.instance;
 
-const mochaSuiteFn =
-  !supportedVersion(process.versions.node) || semver.lt(process.versions.node, '8.5.0') ? describe.skip : describe;
-
-mochaSuiteFn('tracing/graphql', function () {
+const skipNodeVersion = {
```

Comment:
The diff in this file looks big, but not a lot has changed

1. wrap the whole execution to run v14,v15,v16
2. forward GRAPHQL_VERSION to the sub processes